backend/Models/CalendarContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore;
```

Comment:
Why is there a CalendarContext? Also a MovieCOntext?
Will you add a separate context for each entity????
https://docs.microsoft.com/en-us/ef/ef6/fundamentals/working-with-dbcontext
"The recommended way to work with context is to define a class that derives from DbContext and exposes DbSet properties that represent collections of the specified entities in the context"

Reply:
I'll fix it asap